R/tuple.R

```diff
@@ -102,6 +104,18 @@ setMethod("delete", "DatasetTuple", function (x, ...) {
 #' @export
 setMethod("name", "ShojiTuple", vget("name"))
 
+#' @rdname tuple-methods
+#' @export
+setMethod("alias", "ShojiTuple", function(object) vget("alias")(object))
```

Comment:
I'm not sure this makes sense as a ShojiTuple method since only VariableTuples would have them.

Reply:
Ah, of course. I moved these into `VariableTuple` methods for the same effect (without the downside).